packages/@okta/vuepress-theme-prose/const/home.const.js

```diff
@@ -0,0 +1,146 @@
+  export const tileData = [
```

Comment:
Make sure this file stays deleted

Reply:
Remember to remove this file and I will give you an approval @AndriiDovhopolov-okta